README.md

```diff
@@ -4,7 +4,7 @@
 
 ## About
 
-Viav is a minimalist, opinionated, voice channel management bot. To see what it's all about just add it to your server and join a voice channel.
+Viav is a minimalist, opinionated, voice channel management bot. To see what it's all about just add it to your server and join a voice channel. Viav doesn't store any user or server information locally, so you know you can trust it in your guild.
```

Comment:
This wording looks suspicious. I'd use something more like `Viav doesn't store any guild information, user information, messages, nicknames, roles, titles... Viav doesn't store anything. period.`

It should also go on its own line.